lib/lte_uart_dfu/Kconfig

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright (c) 2022 INTELLINIUM <giuliano.franchetto@intellinium.com>
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "LTE UART DFU library"
+
+config LTE_UART_DFU
+	bool "LTE UART DFU library"
+	depends on DFU_TARGET
+	depends on DFU_TARGET_MCUBOOT
+	depends on SERIAL
+	depends on UART_INTERRUPT_DRIVEN
+	select RING_BUFFER
+
+if LTE_UART_DFU
+
+config LTE_UART_DFU_DEV_NAME
```

Comment:
I can't see this being used anywhere? But device names are in the past and no longer allowed in zephyr/ncs (`device_get_binding` is deprecated and scheduled for removal)

Reply:
done